templates/e-commerce/product-listing.amp.html

```diff
@@ -44,13 +44,15 @@ <h2 class="h5 mb2">Categories</h2>
       </div>
     </div>
 
-    <div class="col-12 md-col-7 px2 pt2 pb3 md-px4 md-pt1 md-pb7">
-      <div class="md-commerce-header relative md-flex flex-wrap items-center mx1 md-mx0 md-mb2">
-        <h1 class="h3 mb1 mt2 md-mt2 md-mb2 md-ml0 flex-auto">Products</h1>
-        <div class="inline-block mr1 mb3 md-mb0">
-          {{#categories}} {{> partials/select.snip.html}} {{/categories}}
+    <div class="col-12 md-col-7 pt2 pb3 md-px4 md-pt1 md-pb7">
+      <div class="md-commerce-header relative md-flex flex-wrap items-center md-mx0 md-mb2">
+        <h1 class="h3 mb2 md-mt2 md-mb2 md-ml0 flex-auto px2">Products</h1>
+        <div class="commerce-listing-filters pt2 pb2 mb3 md-mb0">
+          <div class="inline-block mr1 pl2">
+            {{#categories}} {{> partials/select.snip.html}} {{/categories}}
```

Comment:
nit: cant you move the spacing classes to the select and get rid of the wrapper?